rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -128,8 +114,7 @@ def _getBondOffset(self, p1, p2):
     return perp, offsetX, offsetY
 
   def _getOffsetBondPts(self, p1, p2, offsetX, offsetY, lenFrac=None):
-    if not lenFrac:
-      lenFrac = self.drawingOptions.dblBondLengthFrac
+    lenFrac = lenFrac or self.drawingOptions.dblBondLengthFrac
```

Comment:
I personally don't find this more readable and would not use this construct, but it's not worth arguing about